kiwi/volume_manager/btrfs.py

```diff
@@ -347,9 +355,9 @@ def _create_snapshot_info(self, filename):
 
     def _get_subvol_name_from_mountpoint(self, volume_mount):
         subvol_name = '/'.join(volume_mount.mountpoint.split('/')[3:])
-        if self.custom_args['root_is_snapshot']:
-            subvol_name = subvol_name.replace('.snapshots/1/snapshot/', '')
-        return os.path.normpath(subvol_name)
+        if self.toplevel_volume and self.toplevel_volume in subvol_name:
+            subvol_name = subvol_name.replace(self.toplevel_volume, '')
+        return os.path.normpath('/'.join(['@', subvol_name]))
```

Comment:
Hmm, and I wonder why "return os.path.normpath(subvol_name)" has changed to "return os.path.normpath('/'.join(['@', subvol_name]))"

?